.github/workflows/test_accuracy.yml

```diff
@@ -34,18 +34,18 @@ jobs:
         run: |
           source venv/bin/activate
           pytest --data=./data tests/python/accuracy/test_accuracy.py
-          DATA=data pytest --data=./data tests/python/accuracy/test_YOLOv8.py
+          # DATA=data pytest --data=./data tests/python/accuracy/test_YOLOv8.py
```

Comment:
Should we just remove this then?

Reply:
Deleted. YOLO needs to be cleaned up though